src/foraging_gui/Foraging.py

```diff
@@ -4616,6 +4616,7 @@ def _generate_upload_manifest(self,FIP=False):
             if not hasattr(self, 'project_name'):
                 self.project_name = 'Behavior Platform'
 
+            # FIXME: How is this if statement working if it does the exact same thing for each condition?
```

Comment:
I think we had originally planned on triggering different code ocean capsules based on data modalities (FIP), but now we just trigger the same capsule. So we could remove the entire `FIP` variable in this function